quickannotator/db/crud/annotation.py

```diff
@@ -49,6 +62,37 @@ def get_annotation_query(model, scale_factor: float=1.0) -> Query:
     return query
 
 
+def anns_to_feature_collection(annotations: List[models.Annotation]) -> geojson.FeatureCollection:
+    """
+    Converts a list of annotations to a GeoJSON FeatureCollection.
+
+    Args:
+        annotations (List[models.Annotation]): A list of annotation objects.
+
+    Returns:
+        geojson.FeatureCollection: A GeoJSON FeatureCollection containing the annotations.
+    """
+    features = []
+    for annotation in annotations:
+        feature = geojson.Feature(
+            id=annotation.id,
+            geometry=geojson.loads(annotation.polygon),
```

Comment:
is this a place to use e.g., urjson or a faster library?

Reply:
Potentially - I'll leave a TODO here but it might be premature optimization - 
1. We don't know if this operation is the main performance bottleneck
2. Looking at your [blog post](https://andrewjanowczyk.com/serialization-and-storage-of-geojson-for-in-digital-pathology/), there's a ~50% read improvement the native json read method. This is substantial, but not an order of magnitude better

The uncompressed write operation is MUCH faster with orjson than native json. However, we generally don't serialize geojson anywhere (ST_AsGeoJSON returns serialized json by default).

Thoughts?